config/initializers/session_store.rb

```diff
@@ -1,4 +1,6 @@
 # frozen_string_literal: true
 # Be sure to restart your server when you modify this file.
 
-Rails.application.config.session_store :cookie_store, key: "_velum_session"
+Rails.application.config.session_store :cookie_store,
+  key:    "_velum_session",
+  secure: Rails.env.production?
```

Comment:
i also think we don't need to have enhanced security on dev/test environments, especially if it slows down the testing.

and i think this error is not related to the cookie setting, it's flakiness because locally, and in 2/3 ruby versions on travis everything is green